extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json

```diff
@@ -51,6 +51,13 @@
             "condition": "if a conditional step has been processed"
           }
         },
+        "xdm:flowControlID": {
```

Comment:
flow control seems like a product-specific thing, is there a generic term?

Reply:
I will add in the description what a flow control in the Journey means ( the product specific definition ). 
I will comment on the PR after the changes are committed and can be reviewed.

Reply:
and this is an extension, so product-specific stuff is permitted.  OK.